buildchain/buildchain/targets/image.py

```diff
@@ -69,3 +70,8 @@ def uncompressed_filename(self) -> Path:
     def tag(self) -> str:
         """Image tag."""
         return '{img.name}:{img.version}'.format(img=self)
+
+    def save_image(self, save_path: Path) -> Union[TaskError, bool]:
```

Comment:
Why not simply `save`? I mean the class is `Image` so `save_image` doesn't add more info than simply `save`.